go.mod

```diff
@@ -1,6 +1,9 @@
 module github.com/moby/buildkit
 
-go 1.22.0
+go 1.22.7
```

Comment:
opened a  PR for this;

-  https://github.com/containerd/stargz-snapshotter/pull/1942